components/schematics/src/ng-update/update-5.0.0/remove-css-imports-rule.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2019 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Tree } from '@angular-devkit/schematics/src/tree/interface';
+
+import { SchematicContext, Rule } from '@angular-devkit/schematics';
+import { readFileFromTree, getMatchingFilesFromTree } from '../../utils';
+
+const toReplaceSelectors = [
+  {
+    from: '~@dynatrace/angular-components',
+    to: '~@dynatrace/barista-components',
+  },
+  {
+    from: "@import '~@dynatrace/barista-components/style/font-styles';\n",
+    to: '',
+  },
+  {
+    from: "@import '~@dynatrace/angular-components/style/font-styles';\n",
+    to: '',
+  },
+  { from: 'dt-card-actions-spacing', to: 'dt-button-child-button-spacing' },
+  { from: 'default-font', to: 'dt-default-font' },
+  { from: 'h1-font', to: 'dt-h1-font' },
+  { from: 'h2-font', to: 'dt-h2-font' },
+  { from: 'h3-font', to: 'dt-h3-font' },
+  { from: 'code-font', to: 'dt-code-font' },
+  { from: 'label-font', to: 'dt-label-font' },
+];
+
+export function removeCssSelectors(): Rule {
+  return (tree: Tree, _: SchematicContext) => {
+    const files = getMatchingFilesFromTree(tree, filePath =>
+      filePath.endsWith('.scss'),
+    );
+
+    for (const file of files) {
+      let content = readFileFromTree(tree, file);
```

Comment:
I'd suggest adding a initial check if the file includes an import to @dynatrace/angular-components - otherwise we have the risk to change mixins that have a very generic name that others out there might use in their own project

Reply:
we cannot do this in case the cluster is proxies our imports 😂